pyanaconda/payload/rpmostreepayload.py

```diff
@@ -195,32 +152,17 @@ def _install(self, data):
         from gi.repository import OSTree, RpmOstree
```

Comment:
Don't forget to move the imports as well. They can be probably imported on the global level of `installation.py`.